packages/edit-site/src/store/private-actions.js

```diff
@@ -13,15 +13,15 @@ import { store as editorStore } from '@wordpress/editor';
 export const setCanvasMode =
 	( mode ) =>
 	( { registry, dispatch } ) => {
+		const isMediumOrBigger =
+			window.matchMedia( '(min-width: 782px)' ).matches;
 		const switchCanvasMode = () => {
 			registry.batch( () => {
-				const isMediumOrBigger =
-					window.matchMedia( '(min-width: 782px)' ).matches;
 				registry.dispatch( blockEditorStore ).clearSelectedBlock();
 				registry.dispatch( editorStore ).setDeviceType( 'Desktop' );
 				registry
 					.dispatch( blockEditorStore )
-					.__unstableSetEditorMode( 'edit' );
+					.__unstableSetEditorMode( mode );
```

Comment:
I'm not sure this change is correct. There's no "view" mode in the block editor mode.

Reply:
Thanks for catching that. That was a hangover from debugging 🤦🏻 

I'll update quickly